framework/src/org/checkerframework/framework/util/MultiGraphQualifierHierarchy.java

```diff
@@ -664,30 +664,37 @@ assert getTopAnnotation(a1) == getTopAnnotation(a2) :
                 "], a2: " + a2 + " [top: " + getTopAnnotation(a2) + "]";
 
         Set<AnnotationMirror> outset = AnnotationUtils.createAnnotationSet();
-        for (AnnotationMirror a1Super : findSmallestTypes(supertypesMap.get(a1))) {
+        for (AnnotationMirror a1Super : supertypesGraph.get(a1)) {
```

Comment:
The changes in this file seem unrelated to the pull request.  Did you make them or are they from someone else?